meetings/2019/LDM-2019-05-15.md

```diff
@@ -32,8 +32,8 @@ Internally to the member, the preconditions may affect the initial null-state of
 
 These apply to anything that yields output: out parameters, return values, fields, properties, indexers, etc: 
 
-- `[MaybeNull]`: The output may be null, even if the type disallows it 
-- `[NotNull]`: The output will not be null, even if the type allows it 
+- `[MaybeNull]`: The output may be null, even if the type disallows it
+- `[NotNull]`: For outputs (`ref`/`out` parameters, return values), the output will not be null, even if the type allows it. For inputs (by-value/`in` parameters) the value passed is known not to be null when we return.
```

Comment:
What about this usage here

https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/Utilities/Contract.cs#L17